src/stream/src/common/log_store_impl/kv_log_store/buffer.rs

```diff
@@ -206,9 +206,12 @@ impl LogStoreBufferInner {
         }
     }
 
-    fn rewind(&mut self) {
+    fn rewind(&mut self, log_store_rewind_start_epoch: Option<u64>) {
+        let rewind_start_epoch = log_store_rewind_start_epoch.unwrap_or(0);
         while let Some((epoch, item)) = self.consumed_queue.pop_front() {
-            self.unconsumed_queue.push_back((epoch, item));
+            if epoch > rewind_start_epoch {
```

Comment:
Question: when `Some(log_store_rewind_start_epoch)`, does it mean we read from `Exclude(log_store_rewind_start_epoch)`?

Reply:
Yes, it means read data later than log_store_rewind_start_epoch.